src/App.js

```diff
@@ -29,11 +30,12 @@ const App = () =>
   // eslint-disable-next-line implicit-arrow-linebreak
   (
     <Router>
-      {/* <Announcement /> */}
+      <Announcement />
       <Container>
         <Sidebar />
         <Routes>
           <Route exact path="/cars" element={<CarsHome />} />
+          <Route exact path="/myreservations" element={<MyReservations />} />
```

Comment:
I think that we can get rid of those exact attributes, we only directly need them for the root path!